src/render/src/App.svelte

```diff
@@ -206,6 +220,56 @@ async function createCFG(params: Params): Promise<CFG> {
   }
 }
 
+let functionAndCFGMetadata: FunctionAndCFGMetadata = {
+  functionData: { name: "", lineCount: 0 , language : undefined},
+  cfgGraphData: { nodeCount: 0, edgeCount: 0, cyclomaticComplexity: 0 },
+};
+
+function updateCFGMetadata(CFG: CFG) {
+  const nodeCount : number = CFG.graph.order;
+  const edgeCount : number = CFG.graph.size;
+  const cyclomaticComplexity : number = CFG.graph.size - nodeCount + 2; 
+  functionAndCFGMetadata.cfgGraphData = {
+    nodeCount,
+    edgeCount,
+    cyclomaticComplexity,
+  };
+}
+
+function extractFunctionName(func: SyntaxNode, language: Language): string {
```

Comment:
A few notes on this one:
- This function seems generally useful, and not just in the /render page. As such, it should be moved into the `src/control-flow` directory.
- Please add tests for all supported languages and function types. See [`functionNodeTypes`](https://github.com/tmr232/function-graph-overview/blob/181173fcae2b66a2296e85c7a7fe7dd79f5679d5/src/control-flow/cfg.ts#L56) for all supported function types.
- What happens when a function is anonymous?